crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -498,6 +498,10 @@ impl PipelineCache {
         self.pipelines.iter()
     }
 
+    pub fn waiting_pipelines(&self) -> impl Iterator<Item = &CachedPipelineId> {
```

Comment:
Please add some doc comments to this. Also CachedPipelineId should be Copy, let's not return borrows on them when we can return copies instead.

Reply:
Will do. I just followed what was done for the `pipelines()` method, which also do not have doc comments. I will add some to it at the same time.